src/OrchardCore.Modules/OrchardCore.ContentFields/Views/ContentPickerField.Edit.cshtml

```diff
@@ -19,16 +20,16 @@
 <div class="@Orchard.GetWrapperCssClasses("field-wrapper", $"field-wrapper-{uniqueName.HtmlClassify()}")" id="@Html.IdFor(x => x.ContentItemIds)_FieldWrapper">
     <label asp-for="ContentItemIds" class="@Orchard.GetLabelCssClasses()">@Model.PartFieldDefinition.DisplayName()</label>
     <div class="@Orchard.GetEndCssClasses()">
-        <div id="@vueElementId" class="vue-multiselect" data-part="@partName" data-field="@fieldName" data-editor-type="ContentPicker" data-selected-items="@selectedItems" data-search-url="@searchUrl" data-multiple="@multiple">
+        <div id="@vueElementId" class="vue-multiselect" data-part="@partName" data-field="@fieldName" data-editor-type="ContentPicker" data-selected-items="@selectedItems" data-edit-url="@editUrl" data-search-url="@searchUrl" data-multiple="@multiple">
```

Comment:
Why `data-edit` not `data-view` or `data-display`?

Also, you'll need to add a permission check to check if the user can view contentitem before displaying a button that could return 401.

Reply:
For the permission I would need some guidance on how to do this the proper way, because I cant wrap the button with something like 
```
@if (await AuthorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, contentItem)
 {
       <button v-on:click="edit(item)" type="button" class="btn btn-success content-picker-default__list-item__edit"><i class="fa-solid fa-eye fa-sm" aria-hidden="true"></i></button>
}
``` 
Because vue is adding the element to the DOM. Any suggestions how to accomplish the check there?

Reply:
Update the `VueMultiselectItemViewModel` by adding a property called `IsViewable` and do the permission check in the `ContentPickerFieldDisplayDriver` driver then you would populate the URL if the `IsViewable` is true. Also, you would have to do similar check when the user selects a new item using the UI.

Reply:
It should be can't `View` the item.